content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -13,6 +13,7 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 ### Added
 - There's a new method **Get Agents For Transfer** ([Web](/messaging/agent-chat-api/v3.2/#get-agents-for-transfer) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-agents-for-transfer)).
 - There's a new method **Get Customer** ([Web](/messaging/agent-chat-api/v3.2/#get-customer) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-customer)) that returns info about a given Customer.
+- There's a new push **Chat Unfollowed** ([RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#chat-unfollowed)) complementary to `unfollow_chat` method.
```

Comment:
The convention in docs is that we capitalize method names, e. g. **Unfollow Chat**, but pushes and webhooks are lowercase & code-like written: `chat_unfollowed` (remember about `_` in the push link.